draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -536,6 +540,20 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
          (see first_unvalidated_packet_acknowledged in <xref target="sec-QLOG"></xref>).</t>
         </list></t> <!-- End of list of actions -->
 
+        <t>When the flow is controlled using BBR, the careful resume is implemented
+           by setting the pacing rate from the remembered values, with the following
+           precautions:</t>
+           <t><list style="symbols">
+              <t>Careful resume is only activated if the flow is in the Startup state</t>
+              <t>The value of the BBR.bw parameter at the beginning of the next round
+                 is only updated if the current value is lower than the remembered value
+                 of BBR.bw_max.</t>
+              <t>In order to avoid disrupting existing connections, the value of BBR.bw
+                 is only incremented by half the remembered value of BBR.bw_max.</t>
+              <t>The increment is only valid for one round. For the next round, the BBR.bw
```

Comment:
/The increment/This increment/ ?

Reply:
Yes. I am going to rewrite this to be more specific. It would be best to leave BBR.bw alone, and just affect the "pacing rate". This will enforce a stricter separation between the probing action (change the passing rate) and the state of the path (BBR.bw, BBR.bw_max) that should only updated by actual measurements.

Reply:
The "only for one round" is key here. That means the probing time will be limit to one RTT worth of transmission, which goes a long way towards minimizing risks.